tests/unit/developers-serializer.test.js

```diff
@@ -0,0 +1,54 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiSubset = require('chai-subset');
+const _ = require('lodash');
+
+const developersSerializer = appRoot.require('api/v1/serializers/developers-serializer');
+const { getDefinitionProps, testSingleResource, testMultipleResources } = appRoot.require('tests/unit/serializer-test-helpers.js');
```

Comment:
Since this file is in the same directory, you shouldn't need `appRoot` here. Could you remove it here and in other files that do the same thing?